### DISCUSSION TITLE
Show status changes on the issue timeline

### DISCUSSION THREAD

**Discussion by spewu** [community/community#6524] [Projects and Issues] [answered]
When changing the status on an issue (e.g. by moving it to a different column in the Kanban board), it would be really nice it is recorded that change to the issue timeline, along with who made the change.

**Comment by alexcnichols** [comment, marked as answer]
👋🏼 Improvements for bringing the beta to parity with classic projects, and more, are coming, including plugging into the issue timeline! Check out this roadmap item for more and subscribe to be notified if/when things change. https://github.com/github/roadmap/issues/287 :octocat:

**Comment by felixfbecker** [comment]
When having a status like "Proposed" or "Planned", it is very important to have _some_ way to be able to see _who_ marked it as "Proposed" or "Planned" if you need to ask any questions, and also see when it happend. This would be possible if there were timeline events for changing status and other column values.

**Comment by ayhanakay1212** [comment]
Hello, I want to use the State Timeline node. I've encountered a problem. Although the flow is working correctly, I can't see the image on the dashboard. I checked the group settings. I deleted and reinstalled the node. I also reinstalled the State Timeline. The problem persists. It's clear that something happened to the DASHBOARD group to which I added the node because the group size is increasing. I'd appreciate any help. This is the flow I'm testing..[
    {
        "id": "978e1563f23e3a5a",
        "type": "tab",
        "label": "Makine Durumu Çizelgesi",
        "disabled": false,
        "info": "Bu akış, rastgele makine durumlarını (farklı renklerle) bir UI State Timeline üzerinde gösterir."
    },
    {
        "id": "36f3507e35ead036",
        "type": "inject",
        "z": "978e1563f23e3a5a",
        "name": "Her 5 Saniyede Bir",
        "props": [
            {
                "p": "payload"
            }
        ],
        "repeat": "5",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "",
        "payloadType": "str",
        "x": 140,
        "y": 320,
        "wires": [
            [
                "10df50264908c846"
            ]
        ]
    },
    {
        "id": "46e55ddce5d21325",
        "type": "debug",
        "z": "978e1563f23e3a5a",
        "name": "Payload'u Görüntüle",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "payload",
        "targetType": "msg",
        "statusVal": "",
        "statusType": "auto",
        "x": 800,
        "y": 260,
        "wires": []
    },
    {
        "id": "8f4d9dad6ae646fe",
        "type": "ui-state-timeline",
        "z": "978e1563f23e3a5a",
        "group": "da55d39edce2a65f",
        "name": "Makine Durumları",
        "order": 1,
        "width": 0,
        "height": 0,
        "label": "Makine Durumu",
        "barHeight": "6",
        "states": [
            {
                "label": "Arıza",
                "value": "arıza",
                "valueType": "str",
                "color": "#ff0000"
            },
            {
                "label": "Beklemede",
                "value": "beklemede",
                "valueType": "str",
                "color": "#ffff00"
            },
            {
                "label": "Üretimde",
                "value": "üretimde",
                "valueType": "str",
                "color": "#009933"
            },
            {
                "label": "Bakım",
                "value": "bakım",
                "valueType": "str",
                "color": "#0000ff"
            },
            {
                "label": "Hazır",
                "value": "hazır",
                "valueType": "str",
                "color": "#000000"
            }
        ],
        "timelineTimeFormat": "h:mm A",
        "detailsTimeFormat": "M/DD/YY h:mm A",
        "minTimeLabelGap": 6,
        "liveUpdate": true,
        "rangeLimit": 24,
        "rangeLimitUnit": "3600",
        "showTimeMarkers": true,
        "showOnlyStartEndTimes": true,
        "x": 810,
        "y": 320,
        "wires": [
            []
        ]
    },
    {
        "id": "10df50264908c846",
        "type": "function",
        "z": "978e1563f23e3a5a",
        "name": "Rastgele Makine Durumu Oluştur",
        "func": "const states = ['arıza', 'beklemede', 'üretimde', 'bakım', 'hazır'];\nconst randomState = states[Math.floor(Math.random() * states.length)];\n\nmsg.payload = {\n    'state': randomState,\n    'time': Date.now()\n};\n\nreturn msg;",
        "outputs": 1,
        "timeout": 0,
        "noerr": 0,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 460,
        "y": 320,
        "wires": [
            [
                "8f4d9dad6ae646fe",
                "46e55ddce5d21325"
            ]
        ]
    },
    {
        "id": "da55d39edce2a65f",
        "type": "ui-group",
        "name": "TesT",
        "page": "06b57be2c27777cb",
        "width": "17",
        "height": 1,
        "order": 2,
        "showTitle": true,
        "className": "",
        "visible": "true",
        "disabled": "false",
        "groupType": "default"
    },
    {
        "id": "06b57be2c27777cb",
        "type": "ui-page",
        "name": "TEST ALANI",
        "ui": "baf5682286a8da1c",
        "path": "/page7",
        "icon": "bird",
        "layout": "grid",
        "theme": "4eec479671a45853",
        "breakpoints": [
            {
                "name": "Default",
                "px": "0",
                "cols": "3"
            },
            {
                "name": "Tablet",
                "px": "576",
                "cols": "6"
            },
            {
                "name": "Small Desktop",
                "px": "768",
                "cols": "9"
            },
            {
                "name": "Desktop",
                "px": "1024",
                "cols": "12"
            }
        ],
        "order": 6,
        "className": "",
        "visible": "true",
        "disabled": "false"
    },
    {
        "id": "baf5682286a8da1c",
        "type": "ui-base",
        "name": "ATAYLAR MAKINE MİS ",
        "path": "/dashboard",
        "appIcon": "",
        "includeClientData": true,
        "acceptsClientConfig": [
            "ui-notification",
            "ui-control"
        ],
        "showPathInSidebar": false,
        "headerContent": "dashboard",
        "navigationStyle": "default",
        "titleBarStyle": "fixed",
        "showReconnectNotification": false,
        "notificationDisplayTime": 1,
        "showDisconnectNotification": false,
        "allowInstall": true
    },
    {
        "id": "4eec479671a45853",
        "type": "ui-theme",
        "name": "Default Theme",
        "colors": {
            "surface": "#13a7e7",
            "primary": "#e7e4e7",
            "bgPage": "#4ba7b9",
            "groupBg": "#fffafa",
            "groupOutline": "#170202"
        },
        "sizes": {
            "density": "default",
            "pagePadding": "12px",
            "groupGap": "12px",
            "groupBorderRadius": "4px",
            "widgetGap": "12px"
        }
    },
    {
        "id": "1927afc79c6954be",
        "type": "global-config",
        "env": [],
        "modules": {
            "@cgjgh/node-red-dashboard-2-ui-state-timeline": "0.0.11",
            "@flowfuse/node-red-dashboard": "1.26.0"
        }
    }
]
